scripts/compare_warehouse_tables.py

```diff
@@ -0,0 +1,244 @@
+"""Compare the dev_warehouse with the prod warehouse to make sure that changes to code did not break functionality."""
+from datetime import datetime, timedelta
+from google.cloud import bigquery
+from tabulate import tabulate
+import pandas as pd
+import argparse
+import warnings
+
+# TODO Setup For Running The Code:
+# 1. Make sure to update warehouse_dev:
+#   A. Go to https://fcprod-jenkins.dsp-techops.broadinstitute.org
+#   B. Go to the Data Warehouse tab (if applicable)
+#   C. Click the WIP-data-warehouse-python-development job
+#   D. Check if code is pulling from */master branch (Info found in Configure Section, Under Source Code Management -> Branches to build)
+#   E. In the WIP-data-warehouse-python-development job, press Build Now
+# 2. Run "gcloud auth login" with firecloud.org email
+# 3. Run "gcloud auth application-default login" with firecloud.org email
+# 4. Run "gcloud auth application-default set-quota-project broad-dsde-dev"
+
+
+def call_bigquery(bq, env_query, project_name, verbose=False):
+    """Call BigQuery table."""
+    # Start job
+    query = bq.query(env_query)
+    if verbose:
+        print("Starting job.")
+
+    # Gather results
+    results = query.result()
+    if verbose:
+        print("Job finished.")
+
+    # read RowIterator object into pandas df
+    results_df = results.to_dataframe()
+    if verbose:
+        print(results_df)
+    return results_df
+
+
+def get_tables(schema, verbose=False):
+    """Get a list of the tables in warehouse_dev."""
+    # Getting the list of tables
+    dev_tables = list(schema['table_name'])
+
+    # Remove tables not in Prod or too long to parse
+    dev_tables.remove("rawls_entity")
```

Comment:
`rawls_entity` and `rawls_entity_attribute` will likely (hopefully) be included at some point - it might be nice to make a variable like `TABLES_TO_SKIP = ['rawls_entity','rawls_entity_attribute']` and then in this function do a for loop and remove those tables, but also add a comment to the output of the script that notes that these tables were skipped